composer.json

```diff
@@ -16,5 +16,8 @@
         "psr-0": {
             "": "app/code/community"
         }
+    },
+    "extra": {
+        "magento-version-ee": "^1.12.0"
```

Comment:
Does this in itself add the version-checker, it looks like it just specifies that it needs magento enterprise 1.12?